frontend/src/app/Settings/Users/ManageUsersContainer.test.tsx

```diff
@@ -142,7 +143,14 @@ describe("ManageUsersContainer", () => {
   it("loads the component and displays users successfully", async () => {
     const { container } = renderComponentWithMocks(mocks, store);
     await waitForElementToBeRemoved(screen.queryByText("Loading..."));
-    await screen.findByText(/barnes, ben billy/i);
+    await waitFor(() =>
```

Comment:
This block of changes doesn’t appear to be related to the goal of this ticket. Can you give some info on what you did here and why?

Reply:
Got it, thanks for clarifying!